src/pages/OrganizerAddModal.tsx

```diff
@@ -325,6 +325,7 @@ const OrganizerAddModal = ({
           <Title size={2}>Contact</Title>
           <ContactInfoStep
             isOrganizer={true}
+            onValidationChange={() => ({})}
```

Comment:
Why is this necessary?
If it's optional it should be ok right?

Reply:
Also `() => ({})` doesn't match the type specified `(status: ValidationStatus) => void;`

SInce you're returning an object instead of void

Reply:
I worked backward, since it was crashing because the prop was not provided I made it not optional, then provided it in a way that wouldn't crash, but it's probably better to amend the code to make it support the prop not being passed